docs/api/events.rst

```diff
@@ -0,0 +1,1438 @@
+.. SPDX-License-Identifier: MIT
+
+.. currentmodule:: disnake
+
+.. 29 votes for this and only 6 for splited up version. kill me
+.. anyway, you want it, you got it
+.. conservants lmao
+.. i hope everyone will change their mind when see how weird this looks together with
+.. other documentation being split up :P
+
+.. _discord_events:
+
+Events
+======
+
+This section documents events related to the main library.
+
+What are events?
+----------------
```

Comment:
The headers here cause issues and put everything that follows beneath this question in the hierarchy.

Reply:
Done in [`d15ddd7`](https://github.com/DisnakeDev/disnake/pull/722/commits/d15ddd70e20c611070a3445379763a80b07a8c13)